GamesProject/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
Hey, @MarkoAndrew93, please move these files to your folder as stated in point 4 of [task description](https://github.com/MastersAcademy/fe-course-basic-2023/issues/61#issue-1947985307)